Cargo.toml

```diff
@@ -1,3 +1,11 @@
 [workspace]
 members = ["pasture-core", "pasture-io", "pasture-tools", "pasture-derive", "pasture-algorithms"]
 resolver = "2"
+
+[patch.crates-io]
+# needed for emscripten fix
+# https://github.com/chronotope/chrono/pull/568
+chrono = { git = 'https://github.com/chronotope/chrono', branch = 'main' }
+# custom getrandom to enable js feature by default.
+# We don't directly pulll the crate so no idea how to do it otherwise.
+getrandom = {git = 'https://github.com/nyorain/getrandom', branch = 'master' }
```

Comment:
This does not work for me for some reason. What works however is to explicitly include `getrandom` in the `Cargo.toml` of the `pasture-algorithms` module, and enable the `js` feature there.